
Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpr-build-comment[0m and container [32mstep-credential-initializer-6ppmq[0m
{"level":"warn","ts":1580732578.524235,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1580732578.5248332,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpr-build-comment[0m and container [32mstep-working-dir-initializer-vs22z[0m
{"level":"warn","ts":1580732579.5401468,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580732579.5424464,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpr-build-comment[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpr-build-comment[0m and container [32mstep-create-dir-workspace-922hc[0m
{"level":"warn","ts":1580732585.6218562,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580732585.6241796,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpr-build-comment[0m and container [32mstep-git-source-seldonio-seldon-core-pr-1397-li-pkkhm[0m
{"level":"warn","ts":1580732585.9162834,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1580732589.181646,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ master in path /workspace/source"}

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpr-build-comment[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:2911d21cf3142cacf9c82cf40fc3c8e1e166e54f,1397:65e5f01368b45d974bce7f451ff04a16c77081e5
DEBUG: ran git fetch --unshallow origin 65e5f01368b45d974bce7f451ff04a16c77081e5: 2911d21cf3142cacf9c82cf40fc3c8e1e166e54f: in 
DEBUG: ran git checkout master in 
DEBUG: ran git reset --hard 2911d21cf3142cacf9c82cf40fc3c8e1e166e54f in 
DEBUG: ran clean --force -d . in 
DEBUG: ran git merge 65e5f01368b45d974bce7f451ff04a16c77081e5 in 
Merged SHA 13713c836012a4aa27f91b64972939472d14db63 with commit message 'Merge commit '65e5f01368b45d974bce7f451ff04a16c77081e5'' into base branch master
Merged SHA 65e5f01368b45d974bce7f451ff04a16c77081e5 with commit message 'Bump pandas from 0.25.3 to 1.0.0 in /python' into base branch master

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpr-build-comment[0m and container [32mstep-step2[0m
+++ dirname ./add-pr-build-comment
++ cd .
++ pwd
+ STARTUP_DIR=/workspace/source/ci
+ REPO_OWNER=SeldonIO
+ REPO_NAME=seldon-core
+ PULL_NUMBER=1397
+ BUILD_NUMBER=4
+ PIPELINE_CONTEXT=lint
++ cat
+++ date
+ COMMENT_MSG='Mon Feb  3 12:23:15 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1397/4.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1397 --build=4'
+ ** step pr comment --owner=SeldonIO --repository=seldon-core --pull-request=1397 '--comment=Mon Feb  3 12:23:15 UTC 2020
The logs for [lint] [4] will show after the pipeline context has finished.  
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-1397/4.log

impatient try  
** get build logs SeldonIO/seldon-core/PR-1397 --build=4'

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpr-build-comment[0m and container [32mstep-source-mkdir-seldonio-seldon-core-pr-1397-li-shgvs[0m
{"level":"warn","ts":1580732597.7635944,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580732597.768571,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpr-build-comment[0m and container [32mstep-source-copy-seldonio-seldon-core-pr-1397-li-5t6xd[0m
{"level":"warn","ts":1580732598.0438318,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580732598.3623989,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r source/. /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpython-lint-task[0m and container [32mstep-credential-initializer-9v2gm[0m
{"level":"warn","ts":1580732619.119835,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: \"ref: refs/heads/0.8.0-**-support-backwards-incompats\" is not a valid GitHub commit ID"}
{"level":"info","ts":1580732619.1203518,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpython-lint-task[0m and container [32mstep-working-dir-initializer-tl9cf[0m
{"level":"warn","ts":1580732620.077875,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580732620.0808082,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpython-lint-task[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpython-lint-task[0m and container [32mstep-create-dir-workspace-sb684[0m
{"level":"warn","ts":1580732627.3142238,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580732627.3213913,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpython-lint-task[0m and container [32mstep-source-copy-workspace-v82t2[0m
{"level":"warn","ts":1580732627.8711872,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1580732630.515341,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-1397-li-x45nm-4[0m stage [32mpython-lint-task[0m and container [32mstep-python-lint[0m
make: Entering directory '/workspace/source/python'
pip install black==19.3b0
Collecting black==19.3b0
  Downloading https://files.pythonhosted.org/packages/30/62/cf549544a5fe990bbaeca21e9c419501b2de7a701ab0afb377bc81676600/black-19.3b0-py36-none-any.whl (89kB)
Collecting toml>=0.9.4
  Downloading https://files.pythonhosted.org/packages/a2/12/ced7105d2de62fa7c8fb5fce92cc4ce66b57c95fb875e9318dba7f8c5db0/toml-0.10.0-py2.py3-none-any.whl
Collecting attrs>=18.1.0
  Downloading https://files.pythonhosted.org/packages/a2/db/4313ab3be961f7a763066401fb77f7748373b6094076ae2bda2806988af6/attrs-19.3.0-py2.py3-none-any.whl
Collecting click>=6.5
  Downloading https://files.pythonhosted.org/packages/fa/37/45185cb5abbc30d7257104c434fe0b07e5a195a6847506c074527aa599ec/Click-7.0-py2.py3-none-any.whl (81kB)
Collecting appdirs
  Downloading https://files.pythonhosted.org/packages/56/eb/810e700ed1349edde4cbdc1b2a21e28cdf115f9faf263f6bbf8447c1abf3/appdirs-1.4.3-py2.py3-none-any.whl
Installing collected packages: toml, attrs, click, appdirs, black
Successfully installed appdirs-1.4.3 attrs-19.3.0 black-19.3b0 click-7.0 toml-0.10.0
WARNING: You are using pip version 19.3.1; however, version 20.0.2 is available.
You should consider upgrading via the 'pip install --upgrade pip' command.
black \
	--check ./ ../testing \
	--exclude "(testing/scripts/proto|seldon_core/proto/|.eggs|.tox)"
All done! ‚ú® üç∞ ‚ú®
53 files would be left unchanged.
make: Leaving directory '/workspace/source/python'
